Amani-Labs/ams-api
# Setup Circle-CI code-climate and testing framework

#### Description
 - Add Circle-CI integration
 - Add Code Climate integration
 - Set up a testing framework
 - Add a pre-commit hook to run linting script

#### How to Test
- Check for status checks on `pull request`.
- You can run `yarn test` to run all tests.
- Make any changes on ts/js/tsx files and default the set rules and try to commit the changes

#### Any background context you want to add
N/A
#### JIRA

[Finishes AMS-14]
